packages/i18next/src/lib/types.ts

```diff
@@ -121,9 +132,17 @@ export type DiscordChannel = TextBasedDiscordChannel | StoreChannel | StageChann
  * This context enables implementation of per-guild, per-channel, and per-user localization.
  */
 export interface InternationalizationContext {
+	/** The {@link Guild} object for which the language should be fetched, or `null` if the language is to be fetched in a DM. */
 	guild: Guild | null;
+	/** The {@link DiscordChannel} object for which the language should be fetched. */
 	channel: DiscordChannel | null;
-	author: User | null;
+	/**
+	 * @deprecated use {@link InternationalizationContext.user} instead. Will be removed in next major version.
+	 * The user for which the language should be fetched.
+	 */
+	author?: User | null;
+	/** The user for which the language should be fetched. */
+	user: User | null;
```

Comment:
Replace everything here of the form "The X for which the language should be fetched" (passive voice) to "The X to fetch the preferred language for"